src/components/screen-header/HeaderButton.tsx

```diff
@@ -108,6 +110,14 @@ const useHeaderButton = (
     }
     case 'chat':
       return chatIcon;
+    case 'info':
+      const {onPress, type, color, ...accessibilityProps} = buttonProps;
+      return {
+        children: <ThemeIcon svg={Info} colorType={color}/>,
+        accessibilityHint: t(ScreenHeaderTexts.headerButton[type].a11yHint),
```

Comment:
Based on the discussion with @marius-at-atb at the office, since the headerButton hint should not be a specific function (i.e. only tells the generic functionality of the button), I will move this hint and put it into `accessibilityProps` which should make more sense.

Will fix later today and it should be ready on Monday 15 January 2024.

Reply:
An accessibility label just saying "Info" can be added here I think

Reply:
Added label: Info, and moved the hint to the accessibilityProps, also tested and it seems good!